src/app/elements/table-filter/table-filter.component.html

```diff
@@ -287,7 +285,7 @@
                             <inline-editor class="full-width" [placeholder]="col?.name"
                                 [config]="{type:'textarea', size:'sm'}"
                                 *ngIf="(col.type === 'textarea') && col.editable && !entity.constantRow && !notEditableByProperty(entity, col)"
-                                [maxlength]="col.creation ? col.creation.creationLength ? col.creation.creationLength : '100' : '100'"
+                                [maxlength]="col.creation ? (col.creation.creationLength ? col.creation.creationLength : '100') : '100'"
```

Comment:
is it possible to create method getMaxLength in the component class?
and move there col.creation ? (col.creation.creationLength ? col.creation.creationLength : '100') : '100'
then here will be simple  [maxlength]="getMaxLength()"

Reply:
yes, done